### DISCUSSION TITLE
Official template, update it to show icon and change name

### DISCUSSION THREAD

**Discussion by smolinari** [codesandbox/codesandbox-client#6116] [Q&A] [answered]
Hi,

I'm from the Quasar team and I tried to compare the official template files, but just don't see what we need to do to get our Icon showing (again) and to get the "Q" in Quasar capitalized. 

Can someone offer any tips? 

![image](https://user-images.githubusercontent.com/3314957/132627433-01961308-fac1-41d3-994d-49bbc3b96640.png)

Scott

**Comment by smolinari** [comment]
I'm back to trying to make this happen. Can anyone help?

Scott

**Reply by sannek** [reply]
The logo has been fixed [via a PR to the template](https://github.com/quasarframework/quasar-codesandbox/pull/23)
We determine the logo based on the underlying template, and it was set to `node` instead of `quasar` in the `sandbox.config.json`.

I'm not sure what determines the title (I thought it was the `name` field in `package.json`, but looking at other templates that's not it...). Anyway! I'll try manually changing the template title to `Quasar` with a capital, and hope that that sticks. If not, I'll figure out where we decide that!

**Reply by smolinari** [reply]
Hey. @sannek - Awesome! Thanks so much!!! If there is anything I can do to help, please let me know. 

Also, three things. 

Correct me if I am wrong, but the sandbox creation process takes the template from the repo master branch from here:

https://github.com/quasarframework/quasar-codesandbox

The logo now showing in the button for creating the sandbox is actually an old one, which I need to update. Is the codesandbox system taking the logo from somewhere particular? Or do I need to give you the logo?

And lastly, if we wanted to have a direct link to that sandbox creation from the template, what would the URL be?

Thanks again for you help!

Scott

**Reply by sannek** [reply]
Three answers:

- We take the default branch (`dev` in this repo's case) unless otherwise specified in the url
- The component that renders the logo lives [here](https://github.com/codesandbox/codesandbox-client/blob/master/packages/template-icons/src/QuasarIcon.tsx). If you can, the easiest solution would be a PR that changes that component. Otherwise one of us can take care of it. Is it correct that this is the [correct logo](https://github.com/quasarframework/quasar/blob/dev/ui/dev/src/assets/quasar-logo-full.svg) minus the wordmark?
- A direct url would be `https://codesandbox.io/s/github/quasarframework/quasar-codesandbox` to load the default branch, or `https://codesandbox.io/s/github/quasarframework/quasar-codesandbox/tree/master` to load the master branch. The one that shows in our template overview points to the `master` branch.

I just realised that I capitalised only the sandbox created from `dev`, so I'll make sure to do the _actual_ template as well in a minute!

**Comment by smolinari** [comment]
@sannek - This is the new logo. 

![image](https://user-images.githubusercontent.com/3314957/140314490-fad0beee-0d0c-4561-a737-f75a8a1bce08.png)

If you could handle the update, I'd appreciate it. I'm not sure what I'd need to do, as it seems the old logo is an svg?

And thanks for the links. However, if I go to either of them, the app doesn't compile correctly, whereas the button under "Create Sandbox" is now working perfectly. Yay! 😁  

Scott

**Reply by sannek** [reply]
Lovely! As far as I can tell (almost) all the icons we have currently are SVGs so I do think we need an SVG version of it. A little searching in the Quasar repo gave me this one: https://cdn.quasar.dev/logo-v2/svg/logo.svg but that looks like it has black where yours has white?

**Reply by sannek** [reply]
As to why it's not compiling - I have no idea and would have to ask a coworker. On my end the one pointing to `dev` seems to not be working, but the main template pointing to `master` compiles just fine and shows the landing page.

**Reply by smolinari** [reply]
@sannek - Let me ask our design guy about an .svg version of the logo. 

And you are right. The master version URL worked for me now too. Hmm... no idea what happened earlier. 

Scott

**Reply by smolinari** [reply]
@sannek - This is the .svg version. 

https://github.com/quasarframework/quasar-art/blob/master/dist/svg/logo-dark.svg

Scott

**Reply by smolinari** [reply]
@sannek - And one last request. 😊 

When you update the logo, can you also add "by the Quasar Framework Team" below the Title in the button, similar to how its showing here?

![image](https://user-images.githubusercontent.com/3314957/140478765-a376dfa7-2a10-4960-ae8e-5ff5e1cdb420.png)

I think with the logo and this last change, we are done. And again, I appreciate your time and support!

Scott

**Comment by codebryo** [comment, marked as answer]
Is fixed - logo and template association have been updated.

**Reply by smolinari** [reply]
@codebryo - it will need to be deployed, I guess? When will that happen? Just excited this is done and I want to show the team. 😁 

Scott

**Reply by codebryo** [reply]
Should be deployed already -- one caveat though we stumbled over, we can't show a `by Quasar` note as it's a github imported template and that won't allow us atm to show that information. We'll tackle some extra template work down the line.

**Reply by smolinari** [reply]
@codebryo - Just one last minor, itsy bitsy request. 😁 Would it be possible to make the "q" a capital "Q"? 

Scott
